index.html

```diff
@@ -62,7 +62,7 @@ <h1>Improving the Quality of Sleep</h1>
           habits.
         </p>
 
-        <p>
+        <p id="calculate-sleep">
```

Comment:
Hey @debrakayeelliott would you mind refactoring this `id` into a class instead, maybe with a name calculate-sleep-button so it's a bit more descriptive? 


Reason behind this is that IDs are used to identify something on the page so that we can 'use' that later on. For this task we're adding styling to the element rather than an action of sort so we won't be 'using' this element later on in that sense and a class is more appropriate to use here. We'd use an ID if we wanted to add in some JS later on or we wanted to 'link' to this particular paragraph from e.g table of contents that might be on top